scipy/stats/_stats_py.py

```diff
@@ -7054,8 +7058,8 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
     ...                      mean2=12.0, std2=np.sqrt(39.0), nobs2=11)
     Ttest_indResult(statistic=0.9051358093310269, pvalue=0.3751996797581487)
 
-    For comparison, here is the data from which those summary statistics
-    were taken.  With this data, we can compute the same result using
+    For comparison, here is the data from which those summary statistics were
+    taken.  With this data, we can compute the same result using
```

Comment:
```suggestion
    For comparison, here is the data from which those summary statistics
    were taken.  With this data, we can compute the same result using
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.